app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit.php

```diff
@@ -234,15 +234,18 @@ public function getAttributeSetName()
 
     public function getIsConfigured()
     {
-        if ($this->getProduct()->isConfigurable()
-            && !($superAttributes = $this->getProduct()
-                ->getTypeInstance(true)
-                ->getUsedProductAttributeIds($this->getProduct()))
-        ) {
-            $superAttributes = false;
+        $superAttributes = true;
+        $product = $this->getProduct();
+        if ($product->isConfigurable()) {
+            /** @var Mage_Catalog_Model_Product_Type_Configurable $productType */
+            $productType = $product->getTypeInstance(true);
+            $superAttributes = $productType->getUsedProductAttributeIds($this->getProduct());
+            if (!$superAttributes) {
+                $superAttributes = false;
+            }
         }
 
-        return !$this->getProduct()->isConfigurable() || $superAttributes !== false;
+        return !$product->isConfigurable() || $superAttributes !== false;
```

Comment:
Not sure if this is easier to read:

```suggestion
        $product = $this->getProduct();
        if ($product->isConfigurable()) {
            /** @var Mage_Catalog_Model_Product_Type_Configurable $productType */
            $productType = $product->getTypeInstance(true);
            return !empty($productType->getUsedProductAttributeIds($product));
        } else {
            return true;
        }
```

If it is not, line 242 `$superAttributes = $productType->getUsedProductAttributeIds($this->getProduct());` can be shortened to `$superAttributes = $productType->getUsedProductAttributeIds($product);`

Reply:
Used the shortend version (for now).

Reply:
@kiatng i also thought about make it simplier ... eg first ...

```
if (!$product->isConfigurable()) {
    return true;
}
...
```

... but i thought its better to keep changes as simple as possible, I did not expect your careful reviews :+1: :100: 

There is a lot of code that could be made simpler. If you see something like this ... maybe create a PR?